src/components/CommentsList.jsx

```diff
@@ -21,6 +21,23 @@ export default function CommentsList({ articleId }) {
     setCount(count + 2);
   };
 
+  const addComment = (e) => {
+    e.preventDefault();
+    postData(url, {
+      username: `${user.username}`,
+      body: `${comment}`,
+    })
+      .then((response) => response.json())
+      .then(({ comment }) =>
+        setComments((currentComments) => {
+          const newComments = [...currentComments];
+          newComments.push(comment);
```

Comment:
You could improve the UX by adding the new comment at the top of the list. As you have a pattern whereby the comments can be revealed by using 'see more', it takes a while for a user to see their comment if it's hidden which might lead to multiple attempts at posting if they think nothing happened. Consider also adding a success message